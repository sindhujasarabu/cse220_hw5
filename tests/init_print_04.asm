.data
_id: .word 1234567
_credits: .word 5
_name: .asciiz "Th3 quick Br0wn F0x jumps ov3r 13 lazy d0gs @ 7:45 PM!"

.include "p12_tester.asm"
