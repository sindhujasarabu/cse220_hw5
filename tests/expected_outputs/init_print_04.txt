1234567 5 Th3 quick Br0wn F0x jumps ov3r 13 lazy d0gs @ 7:45 PM!